kudos-board-ui/src/components/CreateBoardModal/CreateBoardModal.jsx

```diff
@@ -34,6 +38,8 @@ function CreateBoardModal({ onClose, onBoardCreated}){
         e.preventDefault();
 
         if (!title.trim() || !categoryId.trim()){
+            // this could be more specific as to which one is required...
+            // Could also use the input required attribute
             setError("Title and Category are required")
```

Comment:
Try to give more information to the user than this vague sentence. which is required? You could also use the required attribute in the inputs + also disable the button if either one of these are not provided.